src/frequenz/channels/file_watcher.py

```diff
@@ -16,16 +31,16 @@
 
 
 class EventType(Enum):
-    """Available types of changes to watch for."""
+    """The types of changes that can be observed."""
 
     CREATE = Change.added
-    """A new file was created."""
+    """The file was created."""
 
     MODIFY = Change.modified
-    """An existing file was modified."""
+    """The file was modified."""
 
     DELETE = Change.deleted
-    """An existing file was deleted."""
+    """The file was deleted."""
```

Comment:
We call it `FileWatcher`, but at least on linux, we are able to watch directories as well I think, although I don't know if we have tests, etc.  Maybe we should support that, but probably for a separate issue.

Btw "The types of changes that can be observed" can maybe change to "The types of file events that can be observed"?

Reply:
Well, depends on how pedantic we want to be, but technically in Unix a directory is a type of file, same as you have block devices files, etc. a directory is just a file with an index of other names and which inodes it points to.  I wonder if we shouldn't be more transparent about this being a wrapper on `awatch` and then relay all the specifics to the dependency... But yeah, for another issue.

I'll use "The types of file events that can be observed" :+1: